templates/registration/login.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>Login</h1>
+    <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
```

Comment:
Ensure that the `form` context variable is passed to this template from the view that renders it. This is necessary for the form to be displayed and processed correctly.